dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/WesToilIT.java

```diff
@@ -150,6 +150,107 @@ public void testCancel() throws InterruptedException {
         final boolean isCanceled = waitForWorkflowState(runId, CANCELED_STATE);
 
         assertTrue("The workflow did not cancel in time.", isCanceled);
+    }
+
+    @Test
+    public void testDirectoryAttachment() throws InterruptedException {
+        // These tests pass, and the files provisioned by Toil look correct, but the outputs are not.
+        String[] commandStatementRun = new String[]{ "workflow", "wes", "launch",
+            "--config", TOIL_CONFIG,
+            "--entry", "github.com/dockstore-testing/wes-testing/single-descriptor-nested-input:main",
+            "--json", ResourceHelpers.resourceFilePath("wesIt/w4_1_test.json"),
+            "-a", ResourceHelpers.resourceFilePath("wesIt/w4_nested"),
```

Comment:
use `--attach` in one of these tests just to be thorough

Reply:
Added: https://github.com/dockstore/dockstore-cli/pull/124/commits/4672000da6be9146b15fba12a33e9be4d16fa5c4